compiler/qsc_rca/tests/overrides.rs

```diff
@@ -104,14 +96,12 @@ fn check_rca_for_controlled_h() {
     let package_store_compute_properties = compilation_context.get_compute_properties();
     check_last_statement_compute_properties(
```

Comment:
These tests are technically no longer of overrides, but I do think it would be useful to keep them (and add a few for the properties when compiling with `@Config(Base)`), but could use some help deciding where to put them.

Reply:
I created some tests in callables.rs that replace these ones. The ones that I added are a bit more extensive since they check the compute properties of the whole callable and expand verification to callables for which the controlled specializations differ between `base` and `unrestricted`.